spec/javascripts/student_profile_v2/student_profile_header_spec.js

```diff
@@ -0,0 +1,31 @@
+//= require ./fixtures
+
+describe('StudentProfileHeader', function() {
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+
+  var StudentProfileHeader = window.shared.StudentProfileHeader;
+  var SpecSugar = window.shared.SpecSugar;
+  var Fixtures = window.shared.Fixtures;
+
+  var helpers = {
+    renderInto: function(el, props) {
+      var mergedProps = merge(props || {}, {
+        student: Fixtures.studentProfile.student
+      });
+      return ReactDOM.render(createEl(StudentProfileHeader, mergedProps), el);
+    }
+  };
+
+  SpecSugar.withTestEl('high-level integration tests', function() {
+    it('renders note-taking area', function() {
+      var el = this.testEl;
+      helpers.renderInto(el);
+
+      expect(el).toContainText('Daisy Poppins');
+      expect(el).toContainText('Arthur D Healey');
+      expect($(el).find('a.homeroom-link')).toContainText('102');
```

Comment:
These are some high-level Jasmine tests.  They render the component on the page with fixture data for a student, and then test whether things like the student's name and school are on the page.  You can run Jasmine tests at [http://localhost:3000/teaspoon/default](http://localhost:3000/teaspoon/default) when Rails is running.